processor/src/main/java/org/mapstruct/ap/internal/model/AbstractBaseBuilder.java

```diff
@@ -55,11 +56,21 @@ public B method(Method sourceMethod) {
      *
      * @return See above
      */
-    Assignment createForgedBeanAssignment(SourceRHS sourceRHS, ForgedMethod forgedMethod) {
-        BeanMappingMethod forgedMappingMethod = new BeanMappingMethod.Builder()
-            .forgedMethod( forgedMethod )
-            .mappingContext( ctx )
-            .build();
+    Assignment createForgedAssignment(SourceRHS sourceRHS, ForgedMethod forgedMethod) {
```

Comment:
I renamed the method to match it's new functionality. Is it misleading that we have 2 method with the same name, but different parameters?